doc/source/advanced.rst

```diff
@@ -213,8 +214,8 @@ tuples:
    s + s[:-2]
    s + s[::2]
 
-``reindex`` can be called with another ``MultiIndex``, or even a list or array
-of tuples:
+The :meth:`~DataFrame.reindex` method of Series/DataFrames can be called with
```

Comment:
Can you add backticks: Series/DataFrames --> \`\`Series\`\`/\`\`DataFrame\`\`

Reply:
not contradicting, just curious: couldn't it also be \`\`Series/DataFrame\`\`?

Reply:
I think the difference would whether the "/" gets styled as code or not: "`Series`/`DataFrame`" vs "`Series/DataFrame`".  I could be wrong though, and maybe reST is smarter than markdown in that regard.